easybuild/easyconfigs/p/PSI/README.md

```diff
@@ -0,0 +1,3 @@
+The developers of PSI decide to rename their project to PSI4. The first version is PSI4/1.0.
```

Comment:
you had an issue where the PSI4 rename and versioning was discussed, include that please?